### DISCUSSION TITLE
Get rejected from github education pack

### DISCUSSION THREAD

**Discussion by usefmahmud** [community/community#124498] [GitHub Education] [answered]
### Hi everyone,

I've tried a lot to apply in the github student pack using my educational email with several verifying docs, but it kept rejected.
![image](https://github.com/community/community/assets/129234471/7b57f6cd-bac3-4890-93cd-beddc9b4515c)

**Comment by johan-naizu** [comment]
Make sure the documents you submit show that your still a student there, ie it should include dates.

**Reply by usefmahmud** [reply]
My current level and academic year are already showed on all of them.

**Reply by johan-naizu** [reply]
is your educational institution recognised by Github?

**Reply by johan-naizu** [reply, marked as answer]
you should probably check this out https://github.com/orgs/community/discussions/111352
